### PR TITLE
[#1033] Improving French translations

### PR DESCRIPTION
**Pull request by loicsans**
Seems like we are missing the map section translations.

### CODE REVIEW

File: client/src/translations/fr.json [hunk 1]
```diff
@@ -76,8 +74,6 @@
   "filter_text": "Filtrer texte",
   "filter": "Filtre",
   "flow_application_url": "Flow Application URL",
-  "generate": "Générer",
-  "generate_geopoints": "Générer des données géométriques",
```
Comment (peeb) [inline client/src/translations/fr.json]: Why were these removed? Bad translations?

Comment (loicsans) [reply client/src/translations/fr.json]: Possibly a mistake

**Comment by kardan** [comment]
@loicsans should we just add these two keys back and merge?

**Comment by peeb** [comment]
The maps translations *are* in #1077 so I'm not sure what's happened.

I suggest we merge this PR since the geo translations will make it into develop from 1077. Thoughts? @loicsans @kardan
